pkg/controller/command/route/command.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package route
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/hyperledger/aries-framework-go/pkg/client/route"
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+	"github.com/hyperledger/aries-framework-go/pkg/controller/command"
+)
+
+var logger = log.New("aries-framework/command/route")
+
+// Error codes
+const (
+	// InvalidRequestErrorCode for invalid requests
+	InvalidRequestErrorCode = command.Code(iota + command.ROUTE)
+
+	// ResponseWriteErrorCode for response write error
+	ResponseWriteErrorCode
```

Comment:
You shouldn't throw error response write, it **may** go in infinite loop. Since error handling again tries to write it to on same writer/response.

Reply:
[Removed](https://github.com/hyperledger/aries-framework-go/compare/e3d2bf8f0eff2b46016f1ba077db6cb3aefac866..2f1b5db8a74a7b7659c94b4d4c346b41e71d5c51#diff-d08f92ab0cfb3a36ef618e710ffc3fedL27)